### PR TITLE
[REM] crm: ACL which hasn't been useful in a while

### PR DESCRIPTION
**Pull request by xmo-odoo**
Only the group_salesman can see the opportunities statbuttons

### CODE REVIEW

**Comment by xmo-odoo** [comment]
@robodoo r+

File: addons/mass_mailing_crm/models/mailing_mailing.py [hunk 1]
```diff
@@ -36,6 +36,8 @@ def _prepare_statistics_email_values(self):
         values = super(MassMailing, self)._prepare_statistics_email_values()
         if not self.user_id:
             return values
+        if not self.env['crm.lead'].check_access_rights('read', raise_exception=False):
```
Comment (tde-banana-odoo) [inline addons/mass_mailing_crm/models/mailing_mailing.py:39]: Actually we are not really leaking information ... Shouldn't we instead compute the field as sudo, as it makes no sense to apply current user ACLs for a statistics email ? We could hide it in view if user does not have the groups (as he cannot do anything with this information, no stat button / no tree view to see), but statistics itself could be computed.

File: addons/test_mass_mailing/tests/test_blacklist_behavior.py [hunk 2]
```diff
@@ -51,8 +51,8 @@ def test_mailing_bounce_w_auto_bl_partner_duplicates(self):
 
     @mute_logger('odoo.addons.mail.models.mail_thread')
     def _test_mailing_bounce_w_auto_bl(self, bounce_base_values):
-        mailing = self.env['mailing.mailing'].browse(self.mailing_bl.ids)
-        target = self.env['mailing.test.blacklist'].browse(self.target_rec.ids)
+        mailing = self.mailing_bl.with_env(self.env)
```
Comment (tde-banana-odoo) [inline addons/test_mass_mailing/tests/test_blacklist_behavior.py:54]: Does that change anything ?

**Review by tde-banana-odoo — COMMENTED** [review]
Zbortch !

PS: don't ask a review then r+, that's a bit rude :)

File: addons/test_mass_mailing/tests/test_blacklist_behavior.py (hunk 2 above)
Comment (xmo-odoo) [reply addons/test_mass_mailing/tests/test_blacklist_behavior.py:54]: Not really, it was a bit of cleaning as I was trying to understand what was occurring, and then I forgot to roll it back.

File: addons/mass_mailing_crm/models/mailing_mailing.py (hunk 1 above)
Comment (xmo-odoo) [reply addons/mass_mailing_crm/models/mailing_mailing.py:39]: Mebbe, out of caution std preferred skipping for now, with the option of modifying it afterwards.

**Pull request closed without merging**